pages/price-feeds/_meta.json

```diff
@@ -42,8 +42,10 @@
   "error-codes": "Error Codes",
   "api-instances-and-providers": "API Instances and Providers",
   "contract-addresses": "Contract Addresses",
-  "examples": "Example Applications",
-
+  "examples": {
+    "title": "Examples ↗",
+    "href": "https://github.com/pyth-network/pyth-examples/tree/main/price_feeds"
+  },
```

Comment:
I think we need an index of the available examples somewhere. I.e., we need something that lets people identify the examples that are relevant to them. Maybe that goes in the readme of the examples repo?

Reply:
err also I think this heading should be "Example Applications" not "Examples". The "Applications" makes it clear what you're going to find over there.

Reply:
I will update the readme of the examples repo